src/eradiate/scenes/measure/_core.py

```diff
@@ -106,13 +147,18 @@ class MeasureSpectralConfig(ABC):
             converter=_measure_spectral_config_srf_converter,
             validator=validators.has_quantity(PhysicalQuantity.DIMENSIONLESS),
         ),
-        doc="Spectral response function. If a path is passed, it attempts "
+        doc="Spectral response function (SRF). If a path is passed, it attempts "
         "to load a dataset from that location. If a keyword is passed, e.g., "
         "``'sentinel_2a-msi-4'`` it tries to serve the corresponding dataset "
-        "from the Eradiate data store."
+        "from the Eradiate data store. By default, the *prepared* version of "
+        "the SRF is served unless it does not exist in which case the *raw* "
+        "version is served. To request that the raw version is served, append "
+        "``'-raw'`` to the keyword, e.g., ``'sentinel_2a-msi-4-raw'``. "
+        "Note that the prepared SRF provide a better speed versus accuracy "
+        "trade-off, but for the best accuracy, the raw SRF should be used. "
         "Other types will be converted by :data:`.spectrum_factory`.",
         type=".Spectrum",
-        init_type="str or .Spectrum or dict or float",
+        init_type="os.PathLike or str or .Spectrum or dict or float",
```

Comment:
Let's keep it simple:
```suggestion
        init_type="Path or str or .Spectrum or dict or float",
```